src/Install/Installer.php

```diff
@@ -276,6 +279,49 @@ private function awaitingMollieOrderState($languageId)
         return true;
     }
 
+    /**
+     * @param $languageId
+     * @return bool
+     * @throws PrestaShopDatabaseException
+     * @throws PrestaShopException
+     */
+    public function orderCompletedOrderState($languageId)
+    {
+        $stateExists = false;
+        $states = OrderState::getOrderStates((int)$languageId);
+        foreach ($states as $state) {
+            if ($this->module->lang('Completed') === $state['name']) {
+                Configuration::updateValue(Mollie\Config\Config::MOLLIE_STATUS_ORDER_COMPLETED, (int)$state[OrderState::$definition['primary']]);
+                $stateExists = true;
+                break;
+            }
+        }
+        if ($stateExists) {
+            return true;
+        }
+        $orderState = new OrderState();
+        $orderState->send_email = false;
+        $orderState->color = '#3d7d1c';
+        $orderState->hidden = false;
+        $orderState->delivery = false;
+        $orderState->logable = false;
+        $orderState->invoice = false;
+        $orderState->module_name = $this->module->name;
+        $orderState->name = [];
+        $languages = Language::getLanguages(false);
+        foreach ($languages as $language) {
+            $orderState->name[$language['id_lang']] = $this->module->lang('Completed');
+        }
```

Comment:
can you copy our utility we have for copying names to all languages ? Also be aware that this will copy current context translated value, not the english one - is it intended ?